sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/FormRecognizerClientImplBuilder.java

```diff
@@ -5,17 +5,38 @@
 package com.azure.ai.formrecognizer.implementation;
 
 import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpClient;
 import com.azure.core.http.HttpPipeline;
 import com.azure.core.http.HttpPipelineBuilder;
 import com.azure.core.http.policy.CookiePolicy;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLoggingPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.HttpPolicyProviders;
 import com.azure.core.http.policy.RetryPolicy;
 import com.azure.core.http.policy.UserAgentPolicy;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.ServiceVersion;
 import com.azure.core.util.serializer.JacksonAdapter;
 import com.azure.core.util.serializer.SerializerAdapter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 /** A builder for creating a new instance of the FormRecognizerClient type. */
 @ServiceClientBuilder(serviceClients = {FormRecognizerClientImpl.class})
 public final class FormRecognizerClientImplBuilder {
+    private static final String SDK_NAME = "name";
```

Comment:
Is this generated code? should this be Form Recognizer?

Reply:
Yes, this is generated code. And this is updated in the custom builders [here](https://github.com/Azure/azure-sdk-for-java/blob/feature/formrecognizer_v2.1-preview2/sdk/formrecognizer/azure-ai-formrecognizer/src/main/resources/azure-ai-formrecognizer.properties)